deploy/examples/cluster-on-pvc.yaml

```diff
@@ -12,6 +12,11 @@ metadata:
   name: rook-ceph
   namespace: rook-ceph # namespace:cluster
 spec:
+  network:
+    provider: multus
+    selectors:
+      public: rook-ceph/public-net
+      cluster: rook-ceph/cluster-net
```

Comment:
Ignore the changes to the `cluster-on-pvc.yaml` file. They are in a separate commit that I'll remove before final review.